receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper_linux.go

```diff
@@ -35,3 +35,11 @@ func (s *scraper) recordCPUUtilization(now pcommon.Timestamp, cpuUtilization uca
 	s.mb.RecordSystemCPUUtilizationDataPoint(now, cpuUtilization.Steal, cpuUtilization.CPU, metadata.AttributeStateSteal)
 	s.mb.RecordSystemCPUUtilizationDataPoint(now, cpuUtilization.Iowait, cpuUtilization.CPU, metadata.AttributeStateWait)
 }
+
+func (s *scraper) recordCPUPhysicalCountDataPoint(now pcommon.Timestamp, count int) {
+	s.mb.RecordSystemCPUPhysicalCountDataPoint(now, int64(count))
+}
+
+func (s *scraper) recordCPULogicalCountDataPoint(now pcommon.Timestamp, count int) {
+	s.mb.RecordSystemCPULogicalCountDataPoint(now, int64(count))
+}
```

Comment:
Why do we need to extract these functions if there are identical for linux and others? We can just use `s.mb.RecordSystemCPUPhysicalCountDataPoint(now, int64(count))` in cpu_scraper.go